source/beamline/BioXAS/BioXASZebra.cpp

```diff
@@ -131,8 +146,12 @@ void BioXASZebra::onConnectedChanged()
 	foreach (AMControl *orBlock, orBlocks_)
 		connected &= orBlock->isConnected();
 
+	foreach (AMControl *outputControl, outputControls_)
+		connected &= outputControl->isConnected();
```

Comment:
That would simplify some things, for sure. Right now `BioXASZebraPulseControl` does not inherit from AMControl, so we'd have to refactor that first.